src/plotman/archive.py

```diff
@@ -15,6 +17,13 @@
 
 # TODO : write-protect and delete-protect archived plots
 
+jobs = []
```

Comment:
are these also archive jobs?  it's unclear...  in the rest of the code base, "jobs" on its own always refers to plotting jobs.

Reply:
will make this more descriptive. doing some testing right now.